Tests/Unit/Repository/PhpcrOdmRepositoryTest.php

```diff
@@ -138,4 +143,98 @@ public function testGetVersionsWillThrow()
 
         $this->getRepository()->getVersions('/test');
     }
+
+    /**
+     * @dataProvider provideAddInvalid
+     *
+     * @expectedException \InvalidArgumentException
```

Comment:
Wouldbe nice to check an important part of the message, otherwise we do not know if the Exception that is thrown is the one we expected, e.g.

```
@expectedExceptionMessage invalid parameters
```